complete/pom.xml

```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.springframework</groupId>
```

Comment:
These three tags are used by maven to create and name your jar when you make a build.   

https://maven.apache.org/guides/mini/guide-naming-conventions.html

You want your groupId  to match with your package name in src/main/java.  So you could do something like org.galatea.jen as your group id and change your package structure from src/main/java/price/checker  to src/main/java/org/galatea/jen

Reply:
The groupId, artifactId, version tags have been changed and are now in-line with the naming conventions specified by Maven.

<modelVersion>4.0.0</modelVersion>
<groupId>org.galatea.jen</groupId>
<artifactId>galatea-starter-jen</artifactId>
<version>0.0.0-SNAPSHOT</version>


I have not changed the groupId, artifactId, and version tags within the parent tag because changing any of the tags to something different incurs an error. 

https://maven.apache.org/guides/introduction/introduction-to-the-pom.html#Example_1